packages/fake-driver/lib/logger.js

```diff
@@ -1,4 +1,4 @@
-import { logger } from 'appium-support';
+import { logger } from '@appium/support';
```

Comment:
I am not entirely sure why this wasn't caught before.

Reply:
probably some other package still had appium-support as a dep?